benchmark/benchmark_roberta_model.py

```diff
@@ -0,0 +1,64 @@
+from argparse import ArgumentParser
+
+import torch
+import torchtext
+from benchmark.utils import Timer
+from torchtext.functional import to_tensor
+
+
+def basic_model_input(encoder):
+    transform = encoder.transform()
+    input_batch = ["Hello world", "How are you!"]
+    return to_tensor(transform(input_batch), padding_value=1)
+
+
+def _train(model, model_input):
+    model_out = model(model_input)
+    model_out.backward(torch.ones_like(model_out))
+    model.zero_grad()
+
+
+def run(args):
+    encoder = args.encoder
+    num_passes = args.num_passes
+    warmup_passes = args.num_passes
+    model_input = args.model_input
+
+    if encoder == "xlmr_base":
+        encoder = torchtext.models.XLMR_BASE_ENCODER
+    elif encoder == "xlmr_large":
+        encoder = torchtext.models.XLMR_LARGE_ENCODER
+    elif encoder == "roberta_base":
+        encoder = torchtext.models.ROBERTA_BASE_ENCODER
+    elif encoder == "roberta_large":
+        encoder = torchtext.models.ROBERTA_LARGE_ENCODER
```

Comment:
Add a `raise` in case the string `encoder` is not recognized, otherwise line 36 will fail.